output: apply to change `extract_placeholders()` API behavior (#630)
fluent/fluentd-docs-gitbook: configuration/buffer-section.md

</match>
```

### Path Boundary Validation

> Since v1.19.3, Fluentd validates the values substituted into `${tag}` and `${key}` placeholders so that they cannot cross an unintended path boundary. Because tags and record fields come from incoming events, an unexpected event could otherwise make a plugin like `out_file` create or access a file outside of the intended directory.

Reviewer: ```suggestion
Since v1.19.3, Fluentd validates the values substituted into `${tag}` and `${key}` placeholders to prevent unintended path traversal. Because tags and record fields come from incoming events, an unexpected event could otherwise make a plugin like `out_file` create or access a file outside of the intended directory.
```

How about using known terms here?
